### PR TITLE
Fixes for external links

### PR DESCRIPTION
**Pull request by vitalkanev**
### "Normal" fixes
- Try to resolve an issue with plus4world dashes (only releaser for now)

### Additions
- defacto2.net (self-explainatory..?)
- events.retroscene.org (basically a "database" of Russian demoparties. Demozoo has links to other Russian sites such as SpeccyWiki)

### CODE REVIEW

File: demoscene/utils/groklinks.py [hunk 1]
```diff
@@ -1568,13 +1569,13 @@ class RetroSceneEntry(BaseUrl): # for use as an abstract superclass
...
     tests = [
-        regex_match(r'https?://events\.retroscene\.org/([\w-]+)', re.I)
+        regex_match(r'https?://events\.retroscene\.org/([\w-]+)', re.I),
     ]
 
 
 class RetroSceneRelease(RetroSceneEntry):
     def match_retroscene_release(urlstring, url):
-        regex = re.compile(r'https?://events\.retroscene\.org/([\w-]+)/([\w-]+)/([\w-]+)', re.I)
+        regex = re.compile(r'https?://events\.retroscene\.org/([\w-]+)/([\w-]+)/([\w-]+)', re.I),
```
Comment (gasman) [inline demoscene/utils/groklinks.py:1578]: There shouldn't be a comma here - this isn't in a list :-)

File: demoscene/utils/groklinks.py [hunk 2]
```diff
@@ -1531,12 +1531,70 @@ def __unicode__(self):
         return u"https://%s.bandcamp.com/track/%s" % (domain, name)
 
 
+class Defactro2Entry(BaseUrl): # for use as an abstract superclass
```
Comment (gasman) [inline demoscene/utils/groklinks.py:1534]: Typo here - 'Defactro' should be 'Defacto'

File: demoscene/utils/groklinks.py [hunk 3]
```diff
@@ -1531,12 +1531,70 @@ def __unicode__(self):
...
+
+class RetroSceneRelease(RetroSceneEntry):
+    def match_retroscene_release(urlstring, url):
+        regex = re.compile(r'https?://events\.retroscene\.org/([\w-]+)/([\w-]+)/([\w-]+)', re.I),
+        match = regex.match(urlstrings)
+        if match:
+            event, compoName, entryNumber = match.groups()
+            return "%s/%s/%s" % (event, compoName, entryNumber)
+        
+        tests = [match_retroscene_release]
```
Comment (gasman) [inline demoscene/utils/groklinks.py:1584]: This (and the code below) is indented too much - it's inside the `match_retroscene_release` method, which means it never replaces the `tests` from BaseUrl and ends up matching all links.

File: demoscene/utils/groklinks.py (hunk 2 above)
Comment (Bombe) [reply demoscene/utils/groklinks.py:1534]: A “defactro” is a type of intro that tells it like it is!

**Comment by vitalkanev** [comment]
Outdated + many (unresolvable) merge conflicts.

**Pull request closed without merging**